api_docs/changelog.md

```diff
@@ -20,6 +20,16 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 9.0
 
+**Feature level 240**
+
+* [`GET /messages`](/api/get-message), [`GET /messages/{message_id}`](/api/get-message), `POST /zulip-outgoing-webhook`: Added a new field `archived` to messages, allowing client to mark messages from deactivated streams as archived.
```

Comment:
These lines should be wrapped; I'm surprised `pretter` didn't do it for you, but your editor should have an appropriate feature. (E.g. `emacs` has meta-Q).

Reply:
Yeah this should have not passed the lint test.
Anyway, I fixed it.